scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1025,56 +1026,71 @@ static bool match_one(struct layer_match *match, struct task_struct *p, const ch
 	}
 }
 
-static bool match_layer(struct layer *layer, struct task_struct *p, const char *cgrp_path)
+int match_layer(u32 layer_id, pid_t pid, const char *cgrp_path)
```

Comment:
Out of curiosity, can't the verifier track types across global function boundary? ie. I suppose this doesn't work with the lookups in the caller?